tools/tests/test_backend_datadog.py

```diff
@@ -0,0 +1,132 @@
+# Output backends for sigmac
+# Copyright 2021 Datadog, Inc.
+
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser General Public License for more details.
+
+# You should have received a copy of the GNU Lesser General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+import os
+import yaml
+
+import unittest
+
+from sigma.configuration import SigmaConfiguration
+from sigma.parser.rule import SigmaParser
+from sigma.config.mapping import FieldMapping
+from sigma.backends.datadog import DatadogLogsBackend
+
+
+class TestDatadogBackend(unittest.TestCase):
```

Comment:
The backend is named `DatadogLogsBackend`, what about naming this class `TestDatadogLogsBackend` ? It looks like we want to have one file for all datadog backends with the type of the prodcut in the class name (logs). I think we can do the same thing for tests.